module_cheatsheets/Machine_Learning_Cheatsheet.md

```diff
@@ -0,0 +1,86 @@
+| `ggplot2`              | [`labs()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/labs)        | Labels       | Sets the axis, legend, and plot labels if specified                     |
+| `ggplot2`              | [`theme()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/theme)      | Theme        | Sets the specified non-data elements of a plot (i.e. plot title, legend spacing, text size, etc.)       |
+
+<div style="page-break-after: always;"></div>
+
+### `data.table`, `reshape2`, `RColorBrewer`, `ConsensusClusterPlus`, `ggsignif`
+Documentation for each of these packages can be accessed by clicking the package name in the table below.
+
+| Library/Package                                  | Piece of Code                               | What it's called                               | What it does                                                                                                                                   |
+|-----------------------------------------|---------------------------------------------|-------------------------|------------------------------------------------------------------------|
+| [`data.table`](https://www.rdocumentation.org/packages/data.table/versions/1.11.8)       | [`fread()`](https://www.rdocumentation.org/packages/data.table/versions/1.11.8/topics/fread)    | F read           | Reads in data faster than base R                                                                         |
+| [`reshape2`](https://www.rdocumentation.org/packages/reshape2/versions/1.4.3)             | [`melt()`](https://www.rdocumentation.org/packages/reshape2/versions/1.4.3/topics/melt)         | Melt             | Converts an object into a data.frame                                                                           |
+| [`RColorBrewer`](https://www.rdocumentation.org/packages/RColorBrewer/versions/1.1-2)         | [`brewer.pal()`](https://www.rdocumentation.org/packages/RColorBrewer/versions/1.1-2/topics/RColorBrewer)           | Brewer Pal       | Makes specified color palettes from ColorBrewer available in the R environment             |
+| [`ConsensusClusterPlus`](https://www.rdocumentation.org/packages/ConsensusClusterPlus/versions/1.46.0) | [`ConsensusClusterPlus()`](https://www.rdocumentation.org/packages/ConsensusClusterPlus/versions/1.36.0/topics/ConsensusClusterPlus) | Consensus Clustering             | Finds the consensus across multiple runs of the clustering algorithm                              |
+| [`ggsignif`](https://www.rdocumentation.org/packages/ggsignif/versions/0.4.0)             | [`geom_signif()`](https://www.rdocumentation.org/packages/ggsignif/versions/0.4.0/topics/stat_signif)    | Create Significance Layer   | Computes the significance of the difference between groups and adds this annotation to the plot        |
```

Comment:
```suggestion
| [`ggsignif`](https://www.rdocumentation.org/packages/ggsignif/versions/0.4.0)             | [`geom_signif()`](https://www.rdocumentation.org/packages/ggsignif/versions/0.4.0/topics/stat_signif)    | Create Significance Layer   | Adds significance information to the plot. It can be used to run statistical tests and display the significance information from those tests. We use it differently, in a way that gives us more control, in the notebook.       |
```